spec-integration/test-component/test/fixture.json

```diff
@@ -0,0 +1,22 @@
+{
+  "fixtures": {
+    "success": {
+      "msg": {
+        "headers": {},
+        "body": {}
+      },
+      "cfg": {
+        "calendarId": "{{google_calendar_id}}",
```

Comment:
Is there a `.env` file now for this repo for tests?

Reply:
No, this is just a dummy file that doesn't need anything